backend-api/requirements.md

```diff
@@ -0,0 +1,23 @@
+# Requirements
+
+In this API I want to:
+
+- Manage Pets (dogs, cats, birds, fishes, exotic animals, etc...)
+- Manage Pet Hospitals/Veterinarians
+  - Schedule appointments
+  - Check in/out Pets
+- Publish Pets for adoption
+- Manage owners
+- Blacklist/whitelist owners
+
+## Implementation Ideas
+
+Since these entities have a lot of connections, I would like to use a graph database.
+
+Options:
```

Comment:
@joao-r-reis You can add more techs if you know more of them.

Reply:
[JanusGraph](https://janusgraph.org/), [Azure Cosmos DB](https://docs.microsoft.com/en-us/azure/cosmos-db/graph-introduction), [Amazon Neptune](https://aws.amazon.com/neptune/)

Reply:
I will add those. That way they stay as an option